README.md

```diff
@@ -13,18 +17,37 @@ Bugzilla
   - P5 bugs (would take): https://mzl.la/2hdnQXy
 
 Priorities
-- P1 bugs (Blocker: get someone assigned ASAP: also EMAIL necko@moz whenever we have a new P1 bug)
+- P1 bugs (Blocker: get someone assigned ASAP: let all necko folks know (by email or slack) whenever we have a new P1 bug)
 - P2 bugs (Active--we intend to work on these now or this quarter)
-- P3 bugs (backlog)
+- P3 bugs (backlog). P3 usually means no one will work on it, so we need to be careful
+  - One exception is that we can mark a bug as P3 and also make it block another meta bug.
 - There is no P4--don't use it
+  - This is used by bots.
 - P5 bugs (would take)
 
 Principles
 - P1 bugs MUST have a person assigned
 - Non-necko people can be assigned to necko bugs just fine
-- Bugs filed by wpt-bot should be marked triaged as P5. If something is needed there, someone can ni? one of us.
+- Bugs filed by wpt-bot are marked as P4 and not shown in the triage tool. If something is needed there, someone can ni? one of us
```

Comment:
This will change as per our discussion yesterday. But you can change that in a follow-up PR.